lang/python/python-websocket-client/Makefile

```
@@ -0,0 +1,34 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-websocket-client
+PKG_VERSION:=0.57.0
+PKG_RELEASE:=1
+
+PYPI_NAME:=websocket_client
+PKG_HASH:=d735b91d6d1692a6a181f2a8c9e0238e5f6373356f561bb9dc4c7af36f452010
+
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=Javier Marcet <javier@marcet.info>
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-websocket-client
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=WebSocket client for Python. hybi13 is supported
+  URL:=https://github.com/websocket-client/websocket-client
+  DEPENDS:=+python3-light +python3-setuptools +python3-six
+  VARIANT:=python3
+endef
+
+define Package/python3-websocket-client/description
+  WebSocket client for Python. hybi13 is supported
```

```suggestion
  websocket-client module is WebSocket client for python. This provide the low level APIs for WebSocket. All APIs are the synchronous functions.

  websocket-client supports only hybi-13.
```
(wrap lines as appropriate)